packages/vuetify/src/composables/border-radius.ts

```diff
@@ -26,14 +29,15 @@ export const makeBorderRadiusProps = propsFactory({
   rounded: {
     type: [Boolean, Number, String],
     validator: (v: any) => radius.includes(v),
-  },
+  } as Prop<IRounded>,
 })
 
-// Effect
-export function useBorderRadiusClasses (props: BorderRadiusProps) {
-  const borderRadiusClasses = computed(() => {
+export const borderRadiusClassNames = (
+  props: BorderRadiusProps,
+): string[] => {
+  const classes = []
+  if (props && 'rounded' in props) {
```

Comment:
```suggestion

  if (props && 'rounded' in props) {
```

Reply:
there's an eslint rule for new lines after const/let

Reply:
OK